### DISCUSSION TITLE
please update the ESO Mod "No Interact" with the following changes

### DISCUSSION THREAD

**Discussion by uus-sanct** [Rhyono/CraftStore#31] [General]
Rhyono, if you get a chance could you please update the ESO Mod "No Interact"

Originally Posted by NeuroticPixels
As others have stated below... change this in NoInteract.lua:
From:
```lua
local orgInteract = FISHING_MANAGER.StartInteraction
FISHING_MANAGER.StartInteraction = function(...)
    local _,text,_,_,_,_,_,crime = GetGameCameraInteractableActionInfo()
``` 

To:
```lua
local interactionManager = FISHING_MANAGER or INTERACTIVE_WHEEL_MANAGER
local orgInteract = interactionManager.StartInteraction
interactionManager.StartInteraction = function(...)
    local _,text,_,_,_,_,_,crime = GetGameCameraInteractableActionInfo()
```

**Comment by Rhyono** [comment]
That originated from Baertram if you follow it all of the way back. That fix was necessary for backwards compatibility, but I believe the fix I've put out should suffice.

**Comment by uus-sanct** [comment]
you're right it was Baertram, and thanks I was having a hard time explaining to guildies what to do to manually fix it
